src/helpers/utils.tsx

```diff
@@ -1003,3 +1010,32 @@ export const formatDates = (
   const date = moment(value);
   return date.isValid() && typeof value === 'string' ? date.format(dateFormat) : fallbackText;
 };
+
+/** gets access token or redirects to session info page if session is expired */
+export const getAcessTokenOrRedirect = async () => {
+  // check if user is trying to logout
+  const isLogout = history.location.pathname === LOGOUT_URL;
+  // don't check session state if user is trying to logout
+  const checkSessionExpiry = !isLogout && CHECK_SESSION_EXPIRY_STATUS;
+
+  const sessionExpiredOrToken = getAccessToken(store.getState(), checkSessionExpiry);
+
+  if (sessionExpiredOrToken && sessionExpiredOrToken !== TokenStatus.expired) {
+    return sessionExpiredOrToken;
+  }
+  if (sessionExpiredOrToken === TokenStatus.expired) {
+    try {
+      // refresh token
+      const newAccessToken = await refreshToken(EXPRESS_TOKEN_REFRESH_URL, store.dispatch, {});
```

Comment:
should this be a route or the actual endpoint URL?

Reply:
Have updated